module_auto_update/models/module.py

```diff
@@ -46,36 +51,12 @@ def _store_checksum_installed(self, vals):
             elif vals.get('state') == 'uninstalled':
                 self.write({'checksum_installed': False})
 
-    @api.multi
-    def button_uninstall_cancel(self):
-        return super(
-            Module,
-            self.with_context(retain_checksum_installed=True),
-        ).button_uninstall_cancel()
-
-    @api.multi
-    def button_upgrade_cancel(self):
-        return super(
-            Module,
-            self.with_context(retain_checksum_installed=True),
-        ).button_upgrade_cancel()
```

Comment:
I think you removed too much. The hack was to workaround https://github.com/odoo/odoo/pull/18597, but the code you had here was needed AFAIK...